presto-docker-image/build-remote.sh

```diff
@@ -18,6 +15,19 @@ fi
 PRESTO_VERSION=$1
 PRESTO_LOCATION="https://repo1.maven.org/maven2/io/prestosql/presto-server/${PRESTO_VERSION}/presto-server-${PRESTO_VERSION}.tar.gz"
 CLIENT_LOCATION="https://repo1.maven.org/maven2/io/prestosql/presto-cli/${PRESTO_VERSION}/presto-cli-${PRESTO_VERSION}-executable.jar"
+
+if [[ -d tmp ]]; then
+    rm -r tmp
+fi
+mkdir tmp
```

Comment:
Is it intentional that this isn't cleaned up (but is in the other script)?

Reply:
Nope, I missed the second cleanup; fixed.